src/cogent3/core/new_alignment.py

```diff
@@ -2382,7 +2333,7 @@ def make_unaligned_seqs(
     info: dict = None,
     source: OptPathType = None,
     annotation_db: SupportsFeatures = None,
-    strand: dict[str, int] = None,
+    offset: dict[str, int] = None,
```

Comment:
as this is for many sequences (potentially) make a comment about making the argument and attribute name plural

add a comment about documenting the argument too